draft-ietf-tls-tls13.md

```diff
@@ -5033,7 +5033,7 @@ TLS-compliant application MUST implement the following TLS extensions:
 All implementations MUST send and use these extensions when offering
 applicable features:
 
-  * "supported_versions"   is REQUIRED for all ClientHello messages.
+  * "supported_versions"   is REQUIRED for all ClientHello, ServerHello and HelloRetryRequest messages.
```

Comment:
I was under the impression that this section discusses the requirements for the Client Hello (and how the server uses the CH). SV is mentioned a few lines up, so it should probably not be mentioned again here.

Reply:
@Lekensteyn maybe I misinterpreted it, but then only thing mentioned a few lines up is that this extension is required to be supported, not that ServerHello must have it (before, the server chose it's version using the `ServerHello.version` field, just like in TLS 1.2).

I think you're right about that "this section discusses the requirements for the Client Hello (and how the server uses the CH)", since it states below "Servers receiving a ClientHello which does not conform to these requirements MUST abort the handshake with a "missing_extension" alert."

Reply:
"Implementing the following TLS extensions" implies that both sides must follow the requirements of the linked section of the specification. An implementation cannot claim compliance if it fails to handle the (absence of) SV in SH.

Reply:
@Lekensteyn okay then, thank you. Should I close this pull request myself then or should it be closed by one of the maintainers?

Reply:
@iluxonchik it's just my view, you could close it or let @ekr decide ;)

Reply:
Actually, I think this is an improvement. I just missed it.